nshlib/nsh_prompt.c

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
+ * apps/nshlib/nsh_prompt.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+
+#include "nsh.h"
+
+/****************************************************************************
+ * Preprocessor Macros
+ ****************************************************************************/
+
+#define NSH_PROMPT_SUFFIX_LEN       (sizeof(CONFIG_NSH_PROMPT_SUFFIX) - 1)
+#define NSH_PROMPT_SIZE    (CONFIG_NSH_PROMPT_MAX - NSH_PROMPT_SUFFIX_LEN)
```

Comment:
align or remove NSH_PROMPT_SUFFIX_LEN

Reply:
Okay, that is why I used short names before.